### DISCUSSION TITLE
小狼毫不在相应输入栏内显示预编辑内容

### DISCUSSION THREAD

**Discussion by sojuo** [rime/home#1514] [Q&A]
请问如何使小狼毫输入法不在相应程序内显示预编辑内容，而是在小狼毫自带的候选框内显示预编辑内容？
初以为在用户文件夹的`weasel.custom.yaml`中添入`"inline_preedit": false `即可，后经实证此法不能做到不在相应程序内显示预编辑的内容。
请问有何方法不在程序内显示预编辑内容，而是在小狼毫自带的候选框内显示预编辑内容？

**Comment by heam9** [comment]
可以的啊 , 加patch里, 得`style/inline_preedit`

```
patch:
  style/inline_preedit: false      # 行内显示预编辑区：true；false
```

**Reply by kanwofeiwo** [reply]
还是不行欸

**Reply by yfdyh000** [reply]
> 还是不行欸

可能你没加对

**Reply by kanwofeiwo** [reply]
> > 还是不行欸
> 
> 可能你没加对

# example weasel.custom.yaml
patch:
  #app_options/gvim.exe:  # 程序名字全用小寫字母
  #ascii_mode: true
  "menu/page_size": 6  # 保持原有配置不变
  app_options:
    - name: "Gnome-terminal"  # 程序名称，注意大小写
      ascii_mode: true        # 默认为英文模式
  style/inline_preedit: false      # 行内显示预编辑区：true；false  酱紫,应该加对了,但是部署不生效

**Reply by yfdyh000** [reply]
@kanwofeiwo 缩进也对吗。`\build\weasel.yaml`中搜`inline_preedit`看看部署结果，以及是否特定程序被设置，或者特定程序本身问题。

**Reply by kanwofeiwo** [reply]
缩进也对,build里面是true,我改成false了部署仍然不生效,不是某个程序,所有界面都是这样

**Reply by fxliang** [reply]
改的是build目录下的？？？那不行的

**Reply by kanwofeiwo** [reply]
> 改的是build目录下的？？？那不行的

嘤嘤build也改了,外部custom也改了,刚开始外部改了部署没效果,发现build还是true,然后我就把build改了,还是没用

**Reply by yfdyh000** [reply]
> > 改的是build目录下的？？？那不行的
> 
> 嘤嘤build也改了,外部custom也改了,刚开始外部改了部署没效果,发现build还是true,然后我就把build改了,还是没用

改了外部、重新部署后，“build还是true”，说明没改对，生效以build中显示为准。改动build的内容无效，重新部署时会覆盖掉。
直接改所用或所有`.schema.yaml`试试看，重新部署后看效果、build中是否已正确，custom之后您慢慢研究。.schema.yaml可能在重装/更新软件时被覆盖，弄好后注意备份。

**Reply by kanwofeiwo** [reply]
> > > 改的是build目录下的？？？那不行的
> > 
> > 
> > 嘤嘤build也改了,外部custom也改了,刚开始外部改了部署没效果,发现build还是true,然后我就把build改了,还是没用
> 
> 改了外部、重新部署后，“build还是true”，说明没改对，生效以build中显示为准。改动build的内容无效，重新部署时会覆盖掉。 直接改所用或所有`.schema.yaml`试试看，重新部署后看效果、build中是否已正确，custom之后您慢慢研究。.schema.yaml可能在重装/更新软件时被覆盖，弄好后注意备份。

谢谢
